.env.example

```diff
@@ -16,17 +16,19 @@ VERIFY_URL=https://ecommerceee.onrender.com/
 SEND_EMAIL=<your email address>
 VERIFY_URL=http://localhost:5000/users/verify-email
 FRONTEND_URL=http://localhost:5000
+
+PASSWORD_RESET_URL=http://localhost:5000/users/verify-reset-password
+
+# EMAIL CONFIG
+EMAIL=<contact valentin for email>
```

Comment:
Can you give descriptive names to your variables? According to what they do maybe
Instead of saying "contact someone for this variable"